### DISCUSSION TITLE
Criteria for release

### DISCUSSION THREAD

**Discussion by ajunwalker** [schemacrawler/SchemaCrawler#792] [Q&A]
Hi,

Firstly I want to thank you for always addressing issues so promptly! 

I just wanted to ask what the criteria is for a release is, as there is a feature that I am waiting for in a recent merge of yours.

Thanks again!

**Comment by sualeh** [comment]
@ajunwalker Andrew, I usually release when I have finished adding a new feature, or when I have fixed a bug that I am embarrassed about. Which is to say that I release often. I expect to have a release with a fix for you over the weekend.

**Reply by sualeh** [reply]
@ajunwalker - please use SchemaCrawler v16.17.1
